### DISCUSSION TITLE
Pause the Game Over screen

### DISCUSSION THREAD

**Discussion by ghost** [emoose/OutRun2006Tweaks#245] [Suggestions]
### Describe the feature you'd like to see added

Add  a Continue prompt or slow down or pause the Game Over screen so that the entirety of Last Wave can be played.

**Comment by Sonic-TV** [comment]
I guess once the name entry and the IR screen's implemented, you can play the Last Wave entirely then. FYI, you can play the entire track on SDX if you stay at both screens until both screen timers reaches zero.
